### PR TITLE
Add README for RunGen

### PR DESCRIPTION
**Pull request by steven-johnson**
Add initial documentation for how to use RunGen.

Note to reviewers: Not sure if this belongs as a standalone README, or as part of the main README, or as part of the tutorials, or some combination of all three; suggestions are welcome.

### CODE REVIEW

File: README_rungen.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+way.
+
+```
+# This filter expects a 16-bit RGB image as input, but we're giving it an 8-bit grayscale image:
+$ ./bin/local_laplacian.rungen input=../apps/images/gray.png levels=8 alpha=1 beta=1 local_laplacian=/tmp/out.png
+Warning: Image for Input "input" has 2 dimensions, but this argument requires at least 3 dimensions: adding dummy dimensions of extent 1.
+Warning: Image loaded for argument "input" is type uint8 but this argument expects type uint16; data loss may have occurred.
+```
+
+By default, we try to guess a suitable size for the output image(s), based on
```
Comment (abadams) [inline README_rungen.md]: I don't think bounds query logic can do this. Doesn't rungen just use the input size if the output size is not specified?

**Comment by abadams** [comment]
This adds 7.5mb to the repo for a single new image. Can we make it smaller, skip the example, or use a different 16-bit compressed format?

File: README_rungen.md (hunk 1 above)
Comment (steven-johnson) [reply README_rungen.md]: It guesses based on the input size, yes. The reference to bounds-query logic is misleading: there is a comment in RunGen.cpp suggesting that using the algorithm from AddImageChecks would probably produce a better guess. I'l reword the text.

**Comment by abadams** [comment]
Please squash the history to avoid committing the larger file (e.g. do "git reset --soft master" in the branch)

**Comment by abadams** [comment]
This still seems to have the problematic commit (6e786f59622ebce740e9cfae1de0d00764d7b1f8) as a parent.

**Comment by steven-johnson** [comment]
rebased to master and force-pushed. PTAL.

**Pull request merged**